src/environment.rs

```diff
@@ -233,13 +233,28 @@ pub(crate) fn write_environment_file(
         .parent()
         .expect("There should already be a conda-meta folder");
 
-    std::fs::create_dir_all(parent).into_diagnostic()?;
+    match std::fs::create_dir_all(parent).into_diagnostic() {
+        Ok(_) => {
+            let contents = serde_json::to_string_pretty(&env_file).into_diagnostic()?;
+            match std::fs::write(&path, contents).into_diagnostic() {
+                Ok(_) => {
+                    tracing::debug!("Wrote environment file to: {:?}", path);
+                }
+                Err(e) => tracing::debug!(
+                    "Unable to write environment file to: {:?} => {:?}",
+                    path,
+                    e.root_cause().to_string()
+                ),
+            }
+        }
+        Err(e) => tracing::debug!(
```

Comment:
This should not be ignored, as it should either exist or be able to be created when you are running on an installed environment. As it's used for more than the pixi file.